integration_tests/header.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#set -o errexit ... re-enable when we fix the storage deal bug on Lotus side
+set -x
+
+err_report() {
+    echo "Error on line $1 : $2"
+}
+#FILENAME=`basename $0`
+#trap 'err_report $LINENO $FILENAME' ERR
+
+function finish {
+  kill -9 $DEVNETPID
```

Comment:
shouldn't this be SIGINT instead of SIGKILL, to allow lotus etc to clean up after themselves? or do we not care, since we're in a temporary directory?

Reply:
I don't think we care, since it is a temporary directory, we just want the process to release the socket.